communities.json

```diff
@@ -728,6 +728,14 @@
       "quote": "Discord provides an open and easy-to-moderate community space for the Electron community, allowing us to focus on engaging with our community rather than how to manage a community in tools that aren't explicitly built for that.",
       "inviteCode": "electron",
       "githubUrl": "https://github.com/electron"
+    },
+    {
+      "logo": "jetbrains.png",
+      "title": "JetBrains",
+      "quote": "JetBrains is a cutting-edge software vendor specializing in the creation of intelligent development tools, including IntelliJ IDEA – the leading Java IDE, and the Kotlin programming language.",
```

Comment:
This quote is meant to be a quote about Discord:

![Screen Shot 2020-11-12 at 12 15 22 PM](https://user-images.githubusercontent.com/952783/98991489-bef60300-24e0-11eb-9792-97ff7eb2e9c0.png)

Reply:
You can also choose to omit it if you'd like